lib/engine/devices/builtins/thingengine.builtin.ts

```diff
@@ -226,7 +226,10 @@ export default class MiscellaneousDevice extends Tp.BaseDevice {
         return cap.launchURL(String(url));
     }
 
-    do_alert() {}
+    do_alert() {
+        const now = new Date;
+        return { time: new Tp.Value.Time(now.getHours(), now.getMinutes()) };
```

Comment:
There might be a timezone issue here, because Time objects are timezone-specific (they should always be in the timezone of the user), and Date objects are meant to be timezone-agnostic (getHours() is the timezone of the server, which is not relevant). We need to use something like moment-timezone to get the time in the user's timezone.

This is not the first place where we have this problem, but it will be quite visible with this feature, and we should fix it everywhere.